### ISSUE TITLE
Add schedule type limits when setSequentialXXFraction argument is a double

### ISSUE THREAD

**Issue by joseph-robertson** [NREL/OpenStudio#3699]
On FT?
This would prevent warnings in the err file.

**Comment by shorowit** [comment]
I think this also applies to the AlwaysOffDiscrete, AlwaysOnDiscrete, and AlwaysOnContinuous schedules. Maybe others?

**Comment by jmarrec** [comment]
A better alternative (IMHO) is to correctly define it in ScheduleRegistry.cpp in the Singleton by adding it to `scheduleTypes` and add a method `std::vector<ScheduleTypeKey> ZoneHVACEquipmentList_Impl::getScheduleTypeKeys(const Schedule& schedule) const` method.

That way the correct schedule type limits will be applied automatically on call to `setSequentialXXFractionSchedule` if the schedule passed doesn't not have a schedule type limits (which is the case if you call the double argument method)




```cpp
ScheduleTypeRegistrySingleton::ScheduleTypeRegistrySingleton()
{
  // className, scheduleDisplayName, scheduleRelationshipName, isContinuous, unitType, lowerLimitValue, upperLimitValue;

  const ScheduleType scheduleTypes[] =
  {
     [...]
     {"ZoneHVACEquipmentList","Sequential Cooling Fraction Schedule Name", "sequentialCoolingFractionSchedule",true,"",0.0,1.0},
      [...]
```

**Comment by jmarrec** [comment]
cf #3703

**Comment by jmarrec** [comment]
Fixed via #3703 (merged in #3741)
